pyvips/enums.py

```diff
@@ -288,32 +287,35 @@ class DemandStyle(object):
 
     THINSTRIP (str): demand in thin (typically 1 pixel high) strips
 
+    ANY (str): demand geometry does not matter
+
     """
 
     ERROR = 'error'
     SMALLTILE = 'smalltile'
     FATSTRIP = 'fatstrip'
     THINSTRIP = 'thinstrip'
+    ANY = 'any'
```

Comment:
Ah, `VIPS_DEMAND_STYLE_LAST` was missing, which is why this is being generated now.